changelog/v1.17.0-beta26/envoy-bump-and-test-multivalueheader-fix.yaml

```
@@ -0,0 +1,10 @@
+changelog:
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: solo-io
+    dependencyRepo: envoy-gloo
+    dependencyTag: v1.29.3-patch2
+    issueLink: https://github.com/solo-io/gloo/issues/8867
+    resolvesIssue: false
+    description: >-
+      Bump envoy-gloo to bring us onto 1.29.3-patch2, fixing a bug where Lambdas returning multiValueHeaders with non-string
+      type with `unwrapAsApiGateway` enabled would lead to a 500 response
```

I found the changelog description a little hard to understand. What do you think of this? (Note that I left out the part about the version bump since that should be captured by the `dependency*` fields.)

```suggestion
      Fix a bug when Lambdas that return non-string types with `unwrapAsApiGateway` enabled would lead to a 500 response
```

---

Should we escalate this to a fix to make sure it shows up in the changelog? or are we putting that in the EE?

---

https://github.com/solo-io/gloo/pull/9455/commits/90c197c58e05afeefcf4056651f7a13d4ae4fb6b